include/TPP/Dialect/Xsmm/XsmmOps.td

```diff
@@ -226,4 +256,34 @@ def Xsmm_UnaryDispatchOp : Xsmm_Op<"unary.dispatch",[Pure]> {
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// MatmulDispatchOp
+//===----------------------------------------------------------------------===//
+
+// TODO: all the `flags` should be a list.
+class Xsmm_MatmulAndBrgemmDispatchOp<string mnemonic, list<Trait> traits = []> :
+  Xsmm_Op<mnemonic, !listconcat(traits, [Pure])> {
+    let arguments = (ins DenseI64ArrayAttr:$inputs, 
```

Comment:
Would be good to have an explanation of what the attributes refer to. Also for the other ops.

Reply:
added.